src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/ReInviteUserIntegrationTest.java

```diff
@@ -130,4 +132,25 @@ public void should_return_409_when_reinvited_user_gets_active_in_sidam_but_pendi
             assertThat((String) reInviteUserResponse.get("response_body")).contains(String.format("Resend invite failed as user is already active. Wait for %s minutes for the system to refresh.", syncInterval));
         }
     }
+
+    // should not allow re invite of user who is not in organisation
+    @Test
+    public void should_return_403_when_reinvited_user_not_present_in_organisation() {
+
+        if (resendInviteEnabled) {
+            userProfileCreateUserWireMock(HttpStatus.CREATED);
+
+            OrganisationCreationRequest organisationCreationRequest1 = someMinimalOrganisationRequest().build();
```

Comment:
could you  please add one more for external as-well and tend of the test case add separate it with internal and external scenarios

Reply:
done